ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -957,7 +960,12 @@ abstract class QueryStoreAndAuthDependentIntegrationTest
       val numContracts: Long = 2000
       val helperId = domain.ContractTypeId.Template(None, "Account", "Helper")
       val payload = recordFromFields(ShRecord(owner = v.Value.Sum.Party(alice.unwrap)))
-      val createCmd: domain.CreateAndExerciseCommand[v.Record, v.Value, OptionalPkg] =
+      val createCmd: domain.CreateAndExerciseCommand[
+        v.Record,
+        v.Value,
+        domain.ContractTypeId.Template.OptionalPkg,
+        domain.ContractTypeId.OptionalPkg,
+      ] =
```

Comment:
```suggestion
      val createCmd: domain.CreateAndExerciseCommand.LAVUnresolved =
```

With `type LAVUnresolved` added with this expansion.